tests/api.suite.dist.yml

```diff
@@ -5,3 +5,8 @@ modules:
       - REST:
           depends: PhpBrowser
           url: http://localhost/tests/joomla-cms3/
+      - SOAP:
```

Comment:
yes, that is a second part.

Still this second part I was thinking on doing it for B2B directly to test all the soap webservices because for now in redCORE we do not many more webservices included other than contacts in frontend and users.